Source/WebKit/StorageProcess/StorageProcess.cpp

```diff
@@ -310,11 +310,13 @@ void StorageProcess::fetchWebsiteData(PAL::SessionID sessionID, OptionSet<Websit
 {
     auto websiteData = std::make_unique<WebsiteData>();
     WebsiteData* rawWebsiteData = websiteData.get();
+    UNUSED_PARAM(rawWebsiteData);
     auto callbackAggregator = CallbackAggregator::create([this, websiteData = WTFMove(websiteData), callbackID]() {
         parentProcessConnection()->send(Messages::StorageProcessProxy::DidFetchWebsiteData(callbackID, *websiteData), 0);
     });
 
     String path;
+    UNUSED_PARAM(path);
```

Comment:
Both rawWebsiteData and path are used when SERVICE_WORKER or INDEXED_DATABASE are enabled, so they should only be marked as unused in the rest of the cases.

Reply:
done
path will only be declared if either SERVICE_WORKER or INDEXED_DATABASE are enabled.
If not, then rawWebsiteData will be marked as unused